engine/src/systems/lights/PointLightsSystem.cpp

```
@@ -19,21 +19,30 @@
 #include "ecs/Coordinator.hpp"
 
 namespace nexo::system {
-	void PointLightsSystem::update() const
+	void PointLightsSystem::update()
```

Cppcheck (2.10-2)

[style] The function 'update' is never used.

(unusedFunction)